src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/MobEventsUtils.java

```diff
@@ -55,31 +53,43 @@ private MobEventsUtils()
 
     public static void barbarianEvent(final World world, final Colony colony)
     {
-        if(world == null)
+        if(world == null || !colony.isCanHaveBarbEvents())
         {
             return;
         }
 
+        System.out.println("debug1");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS106)